prisma/schema.prisma

```diff
@@ -0,0 +1,36 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+enum BookStatus {
+  NEW
+  VERIFIED
+  BANNED
+  DISABLED
+  PUBLISHED
+}
+
+model Book {
```

Comment:
Book should have description, cover image, reprint, number of paper,...
Please, think about a book instead of making a normal object.